guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,7 +4,8 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+Remote installation of the Katello database on systems where root access is not available is now possible.
```

Comment:
We always add headers, so something like
```suggestion

=== postgresql-evr extension no longer required

Installation of the Katello database on remote systems where root access is not available is now possible.
```

And perhaps also note that you only need a basic PostgreSQL installation now, which means it probably also works with managed databases like Amazon RDS and others.